oid_validation/pulp/oid_validation/oid_validation.py

```diff
@@ -29,14 +30,33 @@ def authenticate(environ, config=None):
     '''
     Framework hook method.
     '''
-    cert_pem = environ["mod_ssl.var_lookup"]("SSL_CLIENT_CERT")
+    # An output stream (file-like object) to which error output can be written.
+    # For many servers, this will output to the main error log. For more
+    # information, see PEP333.
+    wsgi_error_logger = environ["wsgi.errors"].write
 
     if config is None:
         config = _config()
 
+    # Attempt to retrieve the client certificate, and if it isn't available, reject.
+    cert_pem = ''
+    if 'SSL_CLIENT_CERT' in environ:
+        cert_pem = environ['SSL_CLIENT_CERT']
+    if not cert_pem and 'mod_ssl.var_lookup' in environ:
```

Comment:
I guess you would also need to adjust the first if to be ```if 'SSL_CLIENT_CERT' in environ and environ['SSL_CLIENT_CERT']``` which I guess doesn't save you much typing.